contracts/Token.sol

```diff
@@ -4,18 +4,23 @@ import "@openzeppelin/contracts/math/SafeMath.sol";
 // Interfaces and Contracts.
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/upgrades/contracts/Initializable.sol";
+import "./interfaces/IERC1404.sol";
 import "./interfaces/IToken.sol";
 import "./lib/AccountLib.sol";
 import "./Registry.sol";
 import "./Transact.sol";
 
 
-contract Token is Initializable, IToken, IERC20 {
+contract Token is Initializable, IToken, IERC20, IERC1404 {
```

Comment:
@afmfe-iul This is the main change from a functional perspective. Our Token contract now has to follow that `IERC1404` interface and implement the methods it imposes.